docs/pages/admin-guides/deploy-a-cluster/license.mdx

```diff
@@ -151,7 +151,8 @@ Teleport Web UI users will see an alert banner:
 
 ![License expired banner](../../../img/enterprise/license-expired.png)
 
-Attempts to authenticate to Teleport via Single Sign-On connectors will fail.
+Enterprise features like Single Sign-On authentication with SAML/OIDC
```

Comment:
I still think this is unnecessarily verbose.

General rule of thumb, if you can remove part of the sentence and it remains just as clear as it was before, then you probably didn't need the extra words.

_Enterprise features like Single Sign-On will fail_ is fine. We don't need to mention SAML/OIDC connectors.

Reply:
please see update